notation.go

```diff
@@ -325,3 +326,20 @@ func generateAnnotations(signerInfo *signature.SignerInfo) (map[string]string, e
 		annotationX509ChainThumbprint: string(val),
 	}, nil
 }
+
+// GetDescriptorFromPayload parses a signature payload and returns the descriptor
+// that was signed.
+func GetDescriptorFromPayload(payload *signature.Payload) (*ocispec.Descriptor, error) {
```

Comment:
The word `Payload` in `DescriptorFromPayload` is too broad. Consider `DescriptorFromSignaturePayload`.